packages/vue-app/template/server.js

```diff
@@ -58,8 +57,8 @@ const createNext = ssrContext => (opts) => {
     ssrContext.nuxt.serverRendered = false
     return
   }
-  opts.query = stringify(opts.query)
-  opts.path = opts.path + (opts.query ? '?' + opts.query : '')
+  opts.path = withQuery(opts.path, opts.query)
+  opts.query = stringifyQuery(opts.query || {})
```

Comment:
Checking `createNext ` it seems we no longer would need stringifing `opts.query` (not used until res.end)

Reply:
That is very satisfying